config/manifests/ics-operator/bases/ocs-operator.clusterserviceversion.yaml

```diff
@@ -15,16 +15,22 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+    - description: StorageClassClaim is the Schema for the storageclassclaims API
+      displayName: Storage Class Claim
+      kind: StorageClassClaim
+      name: storageclassclaims.ocs.openshift.io
+      version: v1alpha1
```

Comment:
we do have storageclassclaim again as we removed it.

Reply:
should be removed now